src/renderer/services/chain/decimal.ts

```
@@ -5,12 +5,14 @@ import { BCH_DECIMAL } from '@xchainjs/xchain-bitcoincash'
 import { BSC_GAS_ASSET_DECIMAL } from '@xchainjs/xchain-bsc'
 import { ADA_DECIMALS } from '@xchainjs/xchain-cardano'
 import { DASH_DECIMAL } from '@xchainjs/xchain-dash'
-import { CACAO_DECIMAL } from '@xchainjs/xchain-mayachain'
+import { CACAO_DECIMAL, MAYA_DECIMAL } from '@xchainjs/xchain-mayachain'
 import { PoolDetail as MayaPoolDetail } from '@xchainjs/xchain-mayamidgard'
 import { PoolDetail } from '@xchainjs/xchain-midgard'
 import { XRD_DECIMAL } from '@xchainjs/xchain-radix'
 import { SOL_DECIMALS } from '@xchainjs/xchain-solana'
 import { isTCYAsset } from '@xchainjs/xchain-thorchain'
+
+import { isMayaAsset } from '../../helpers/assetHelper'
```

ESLint

[error] `../../helpers/assetHelper` import should occur after import of `../../../shared/utils/asset`

(import/order)